doc/en/source/ngsi_ld_api_walkthrough.rst

```diff
@@ -0,0 +1,791 @@
+
+
+Entities
+=========================
+
+Entities are the units for representing objects in the environment, each having some properties of their own, may also have some relationships with others. This is how linked data are formed.
+
+
+Create entities
+------------------------------------------
+
+There are several ways of creating an NGSI-LD Entity on FogFlow Broker:
+
+* When context is provided in the Link header: The context for resolving the payload is given through the Link header.
+* When context is provided in the payload: Context is in the payload itself, there is no need to attach a Link header in the request.
+* When the request payload is already expanded: Some payloads are already expanded using some context.
+
+Curl requests for creating an entity on FogFlow Broker in different ways are given below. All the are the POST requests to FogFlow Broker. Broker returns a response of "201 Created" for a successful creation of a new entity and "409 Conflict" on creating an already existing entity.
+
+**When context is provided in the Link header:**
+
+.. code-block:: console
+
+        curl -iX POST \
+        'http://<Thin_Broker_IP>:8070/ngsi-ld/v1/entities/' \
+        -H 'Content-Type: application/ld+json' \
+        -H 'Accept: application/ld+json' \
+        -H 'Link: <{{link}}>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
+        -d '
+        {
+                "id": "urn:ngsi-ld:Vehicle:A100",
+                "type": "Vehicle",
+                "brandName": {
+                        "type": "Property",
+                        "value": "BMW",
+                        "observedAt": "2017-07-29T12:00:04"
+                },
+                "isParked": {
+                        "type": "Relationship",
+                        "object": "urn:ngsi-ld:OffStreetParking:Downtown",
+                        "observedAt": "2017-07-29T12:00:04",
+                        "providedBy": {
+                                "type": "Relationship",
+                                "object": "urn:ngsi-ld:Person:Bob"
+                        }
+                },
+                "speed": {
+                        "type": "Property",
+                        "value": 81,
+                        "observedAt": "2017-07-29T12:00:04"
+                },
+                "location": {
+                        "type": "GeoProperty",
+                        "value": {
+                                "type": "Point",
+                                "coordinates": [-8.5, 41.2]
+                        }
+                }
+        }'
+
+**When context is provided in the payload:**
+
+.. code-block:: console
+
+	curl -iX POST \
+	'http://<Thin_Broker_IP>:8070/ngsi-ld/v1/entities/' \
+	-H 'Content-Type: application/ld+json' \
+	-H 'Accept: application/ld+json' \
+	-d '
+	{
+		"@context": [{
+			"Vehicle": "http://example.org/vehicle/Vehicle",
```

Comment:
[minor]remove all the context other than etsi, as currently etsi support is there. it creates confusion.